backend/controllers/addUserCaseload.ts

```diff
@@ -29,14 +35,18 @@ const selectCaseloadsFactory = (getUserAssignableCaseloads: any, saveCaseloads:
 
   const post = async (req: Request, res: Response) => {
     const { userId } = req.params
-    const { caseloads } = req.body
+    const { caseloads, assignableCaseloads } = req.body
```

Comment:
The assignableCaseloads are posted in the form - to avoid re-requesting the data (not sure if there is a neater way?)